cores/esp8266/Arduino.h

```diff
@@ -204,7 +204,19 @@ void setup(void);
 void loop(void);
 
 void yield(void);
-void optimistic_yield(uint32_t interval_us);
+
+#ifndef F_CPU
```

Comment:
Is F_CPU ever not defined?  clockCyclesPerMicrosecond() is always defined, and it just uses that constant, no?

Reply:
@earlephilhower that's a bit of jumping the gun for an idea I have to allow a 3rd option for cpu speed: runtime settable. I'm not sure yet if it's feasible

Reply:
The diff presentation here is quite unreadable, unless one remembers exactly what the surrounding code was. I'll have to check later. BUT: I'd strongly like to keep this check for F_CPU in any case, it doesn't hurt and IMHO makes the change and intention so much clearer.